zerver/webhooks/github/view.py

```diff
@@ -381,6 +381,38 @@ def get_status_body(helper: Helper) -> str:
     )
 
 
+def get_locked_or_unlocked_pull_request_body(helper: Helper) -> str:
+    payload = helper.payload
+
+    action = payload["action"]
+
+    message = (
+        "{sender} has locked [PR #{pr_number}]({pr_url}) and limited conversation to collaborators."
```

Comment:
"and limited conversation to collaborators" reads awkwardly to me; can we copy whatever language GitHub uses to describe this situation?

Reply:
Github uses this format - "... locked this as {reason} and limited conversation to collaborators" . How about "{sender} has locked [PR #{pr_number}]({pr_url}) as {reason} and limited conversation to collaborators." . I am using "and limited conversation to collaborators" as it was originally used by github.

Reply:
OK, yeah, if that's what GitHub's using, I think that's a great reason for us to do the same.